S2VX.Game/SongSelection/SongSelectionScreen.cs

```diff
@@ -0,0 +1,37 @@
+﻿using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.UserInterface;
+using osu.Framework.Screens;
+using osuTK;
+using S2VX.Game.Editor;
+
+namespace S2VX.Game.Play {
+    public class SongSelectionScreen : Screen {
+        private static Vector2 InputSize { get; } = new Vector2(100, 30);
```

Comment:
Wouldn't TextSize be a more fitting variable name in this case since the user would not be inputting anything?

Reply:
True, I'll change it to BtnSize tomorrow since it has to do with buttons

Reply:
Merged and updated to BtnSize